easybuild/easyconfigs/p/Perl/Perl-5.24.1-gimkl-2017a.eb

```diff
@@ -0,0 +1,1195 @@
+        'source_tmpl': 'XML-DOM-%s.tar.gz' % xml_dom_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/T/TJ/TJMATHER/'],
+    }),
+    ('XML::DOM::XPath', xml_dom_xpath_version, {
+        'source_tmpl': 'XML-DOM-XPath-%s.tar.gz' % xml_dom_xpath_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/M/MI/MIROD/'],
+    }),
+    ('XML::Parser::PerlSAX', libxml_perl_version, {
+        'source_tmpl': 'libxml-perl-%s.tar.gz' % libxml_perl_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/K/KM/KMACLEOD/'],
+    }),
+    ('XML::Parser', xml_parser_version, {
+        'source_tmpl': 'XML-Parser-%s.tar.gz' % xml_parser_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/T/TO/TODDR/'],
+    }),
+    ('XML::Twig', xml_twig_version, {
+        'source_tmpl': 'XML-Twig-%s.tar.gz' % xml_twig_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/M/MI/MIROD/'],
+    }),
+    ('XML::Writer', xml_writer_version, {
+        'source_tmpl': 'XML-Writer-%s.tar.gz' % xml_writer_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/J/JO/JOSEPHW/'],
+    }),
+    ('Spreadsheet::ParseExcel', spreadsheet_parseexcel_version, {
+        'source_tmpl': 'Spreadsheet-ParseExcel-%s.tar.gz' % spreadsheet_parseexcel_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/D/DO/DOUGW/'],
+    }),
+    ('Convert::Binary::C', convert_binary_c_version, {
+        'source_tmpl': 'Convert-Binary-C-%s.tar.gz' % convert_binary_c_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/M/MH/MHX/'],
+    }),
+    ('Sys::SigAction.pm', sys_sigaction_version, {
+        'source_tmpl': 'Sys-SigAction-%s.tar.gz' % sys_sigaction_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/L/LB/LBAXTER/'],
+    }),
+    ('Acme::Damn', acme_damn_version, {
+        'source_tmpl': 'Acme-Damn-%s.tar.gz' % acme_damn_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/I/IB/IBB/'],
+    }),
+    ('Set::IntervalTree', set_intervaltree_version, {
+        'source_tmpl': 'Set-IntervalTree-%s.tar.gz' % set_intervaltree_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/B/BE/BENBOOTH/'],
+    }),
+    ('HTML::TableExtract', html_tableextract_version, {
+        'source_tmpl': 'HTML-TableExtract-%s.tar.gz' % html_tableextract_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/M/MS/MSISK/'],
+    }),
+    ('Bio::Perl', bioperl_version, {
+        'source_tmpl': 'BioPerl-%s.tar.gz' % bioperl_version,        
+        'source_urls': ['http://www.cpan.org/modules/by-module/Bio/CJFIELDS/'],
+    }),
+    ('Bio::DB::Sam', bio_samtools_version, {
+        'source_tmpl': 'Bio-SamTools-%s.tar.gz' % bio_samtools_version,
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/L/LD/LDS/'],
+        'patches': ['Perl_SamTools_include.patch'],
```

Comment:
please include version in the patch name, I'd go with `Bio-DB-Sam-1.43_include.patch`